template.md

```diff
@@ -103,7 +105,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
 ### Schedule constraints
 
-_Schedule constraints should be supplied here **48 hours** before the meeting begins so that the Chairs can take them into account when preparing the schedule._
+*Schedule constraints should be supplied here **48 hours** before the meeting begins so that the Chairs can take them into account when preparing the schedule.*
```

Comment:
why is use of the underscore an issue?

Reply:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md049---emphasis-style-should-be-consistent

Reply:
not sure i buy the reasoning, but either way if we want to enforce markdownlint shouldn't we be doing so?

Reply:
certainly could.  we did it in `how-we-work`, so why not?